docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -18,16 +18,20 @@ When your page launches a new window, the test automatically continues in the ne
 ```js
 import { Selector } from 'testcafe';
 
-fixture `Login page`
-    .page('https://login.wrike.com/login/');
-
-const googleButton = Selector('div.login-panel-footer__login-with > button');
-
-test('Login via Google', async t => {
-    await t
-        .click(googleButton)
-        .typeText('input[type=email]', 'This text will be entered inside the login dialog');
+fixture `Microsoft DevBlogs Login page`
+    .page('https://devblogs.microsoft.com/');
+
+test('automatically change execution context', async t => {
+   await t
+    .click('.login-but') #click the login button
+    .click('.nsl-button-google') #open Google's OAuth window
```

Comment:
Need to use the JavaScript comment  here and below
`// open Google's OAuth window`